scrap-point-of-interrest.py

```diff
@@ -0,0 +1,296 @@
+from PIL import Image, ImageOps

```

Comment:
How do you test such a script? It is not included in coverage.

Reply:
I don't

Reply:
We should probably talk to the coaches about this. Is this even allowed? I do not know even if the use of such a script is quite clear.

Reply:
Ok, let me just make a point clear. What is the purpose of testing ? The purpose of testing is to catch issues within your code that is deployed in your client. The goal of this script is to published challenge corresponding with point-of-interest. To "test" it you can just check the output it has generated. Wanted to testing everything just because we "need to" is shear stupidity. This script is not going to get updated and even if it is, its small size make it enough to just read it.

Reply:
ok